src/test/java/org/assertj/core/presentation/StandardRepresentation_toStringOf_Test.java

```diff
@@ -12,8 +12,6 @@
  */
 package org.assertj.core.presentation;
 
-import static java.lang.String.format;
```

Comment:
I usually try to avoid commits with unnecessary imports moves, but this one sneaked on me...